src/main/external-resources/renderers/Panasonic-VieraTXL32V10E.conf

```diff
@@ -41,9 +41,11 @@ Supported = f:mpegps     v:mpeg2|mp4|h264   a:ac3|lpcm|aac|mpa   m:video/mpeg
 Supported = f:avi|divx   v:mp4|divx         a:mp3|ac3            m:video/divx   qpel:yes|no   gmc:0
 
 # Supported image formats:
-Supported = f:jpg   m:image/jpeg
-Supported = f:png   m:image/png
-Supported = f:gif   m:image/gif
+Supported = f:jpg     	m:image/jpeg
+Supported = f:png		m:image/png
+Supported = f:gif		m:image/gif
+Supported = f:tiff		m:image/tiff
+Supported = f:bmp		m:image/bmp
 
```

Comment:
@SubJunk when my old Panasonic TV supports that not documented images formats  what about to add this setting to the other Panasonic TVs confs? Users can test it and we can remove the not working formats for particular renderers.